packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -302,7 +348,8 @@ export abstract class AbstractVSXExtensionComponent extends React.Component<Abst
             return <button className="theia-button action prominent theia-mod-disabled">Installing</button>;
         }
         if (installed) {
-            return <button className="theia-button action" onClick={this.uninstall}>Uninstall</button>;
+            return <div><button className="theia-button action" onClick={this.uninstall}>Uninstall</button>
+                <div className="codicon codicon-settings-gear action" onClick={e => this.manage(e, this.props.extension)}></div></div>;
```

Comment:
@colin-grant-work do you have an opinion of how we bind the event handler here?
I understand it may not be performant, and will be re-rendered:
- https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#no-bind-fn-in-event-handlers

Do you have a strategy to fix it?

Reply:
Hm... so there are a number of things like this in the repo (a number of tree widget event handlers, e.g.), really, so I'm not sure we need to worry too much about it. But my usual strategy for handling this sort of scenario is to give the target something like a `data-id`, and then in the handler, you can use the `data-id` property to retrieve the extension, and then operate on the extension as you need to.